lib/comm_common.cpp

```diff
@@ -308,6 +308,8 @@ bool comm_peer2peer_enabled(int dir, int dim){
   return enable_p2p ? peer2peer_enabled[dir][dim] : false;
 }
 
+bool comm_peer2peer_possible(int dir, int dim) { return peer2peer_enabled[dir][dim]; }
```

Comment:
Might be a dumb question but what's the purpose of this function?

Reply:
I needed it at some point. Basically nvshmem does not care about whether we enable P2P via the env variable. But we exploit the knowledge that we can directly pack into the remote buffer with nvshmem when p2p is possible.
Now that I think of it again there is a potential we don't need that function.

Reply:
I have removed this now.